rules/manager_test.go

```diff
@@ -47,6 +47,10 @@ func TestMain(m *testing.M) {
 	goleak.VerifyTestMain(m)
 }
 
+func suiteEngineQueryFunc(suite *promql.Test) QueryFunc {
```

Comment:
You don't need that if we get promql.Engine outside of Test

Reply:
:heavy_check_mark: